pkg/harvester/components/SettingList.vue

```diff
@@ -112,6 +112,9 @@ export default {
             <span v-if="setting.customized" class="modified">
               Modified
             </span>
+            <span v-if="setting.technicalPreview" v-clean-tooltip="'Technical Previews allow users to test and evaluate early-access functionality prior to official supported releases'" class="technical-preview">
```

Comment:
Could you help put tooltip text in en-us.yaml? 
Thx

Reply:
Sure, done. I wasn't sure exactly where best to put that string in en-ys.yaml, but given the context I added it under `advancedSettings`. Please LMK if it would be better somewhere else.